### PR TITLE
Spine library added

### PR DESCRIPTION
**Pull request by maxgribov**
Added link to unofficial Spine runtime Swift library.

## Project URL
https://github.com/maxgribov/Spine

## Category
Games

## Description
The library allows you to play animations created in the Spine app (http://esotericsoftware.com).
 
## Why it should be included to `Awesome macOS open source applications ` (optional)
Spine app makes it very convenient to create character animations. But unfortunately there is no official Swift runtime, which makes it impossible to use these animations in SpriteKit. This library allows you to do just that.

## Checklist
- [x] Only one project/change is in this pull request
- [x] Addition in alphabetical order
- [x] Appropriate language icon(s) added if applicable
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 2 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
3117 | [301](https://httpstatuses.com/301) | https://github.com/klauscfhq/tusk redirects to<br>https://github.com/klaussinani/tusk
3395 | [301](https://httpstatuses.com/301) | https://github.com/klauscfhq/ao redirects to<br>https://github.com/klaussinani/ao


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -609,4 +610,4 @@ Thanks to all the people who contribute:
 [ruby_icon]: ./icons/ruby-16.png 'Ruby language.'
 [rust_icon]: ./icons/rust-16.png 'Rust language.'
 [swift_icon]: ./icons/swift-16.png 'Swift language.'
-[type_script_icon]: ./icons/typescript-16.png 'TypeScript language.'
\ No newline at end of file
+[type_script_icon]: ./icons/typescript-16.png 'TypeScript language.'
```
Comment (maxgribov) [inline README.md]: Xcode changes this line automatically for some reason

**Comment by maxgribov** [comment]
`613  [type_script_icon]: ./icons/typescript-16.png 'TypeScript language.'`
Xcode changed this line automatically for some reason.

**Comment by serhii-londar** [comment]
@maxgribov I can't merge it because It's library. This list contains only applications. I think you can add it to [Awesome Swift](https://github.com/matteocrippa/awesome-swift) or [Awesime iOS](https://github.com/vsouza/awesome-ios) lists.

**Pull request closed without merging**